src/sentry/tasks/auto_ongoing_issues.py

```diff
@@ -133,14 +97,15 @@ def schedule_auto_transition_to_ongoing() -> None:
 @retry(on=(OperationalError,))
 @log_error_if_queue_has_items
 def auto_transition_issues_new_to_ongoing(
-    project_ids: List[int],
     first_seen_lte: int,
-    organization_id: int,
```

Comment:
Changing the signature of a task is a risky change.
You have to write a PR that can handle both the old signature and the new signature (probably making the call being keyword args).